core/src/main/java/nl/nn/adapterframework/util/flow/FlowDiagramManager.java

```diff
@@ -248,7 +280,7 @@ private void generateFlowDiagram(String name, String dot, File destination) thro
 		long start = System.currentTimeMillis();
 
 		try (FileOutputStream outputStream = new FileOutputStream(destination)) {
-			generator.generateFlow(name, dot, outputStream);
+			getFlowGenerator().generateFlow(name, dot, outputStream);
```

Comment:
Is het zeker dat het resultaat van getFlowGenerator hier nooit null is?

Reply:
Hij kan zeker null zijn, maar de nullcheck zit in de methodes die deze methode aanroepen... zie regel 261